.gitmodules

```
@@ -1,3 +1,4 @@
-[submodule "backend-privates"]
-	path = backend-privates
+[submodule "src/main/resources/config"]
```

이것만 수정 가능할까요? submodule의 이름이 경로보다는 실제 가리키는 레파지토리 이름이면 좋을 거 같아요
```suggestion
[submodule "backend-privates"]
```